cfgov/v1/models/sublanding_page.py

```diff
@@ -75,10 +75,8 @@ def get_browsefilterable_posts(self, request, limit):
                 filtered_controls.update({id: []})
             form_class = page.get_form_class()
             posts = page.get_page_set(form_class(parent=page, hostname=request.site.hostname), request.site.hostname)
```

Comment:
posts could still return `None`, right?  The later call to `chain(*posts)` still results in an error if it is `None`.